framework/PageToolbar/PageToolbarFilters/ToolbarTextFilter.tsx

```diff
@@ -102,3 +109,83 @@ export function ToolbarTextFilter(props: IToolbarTextFilterProps) {
     </InputGroup>
   );
 }
+
+export function ToolbarSingleTextFilter(props: {
+  id?: string;
+  placeholder?: string;
+  comparison: 'contains' | 'startsWith' | 'endsWith' | 'equals';
+  setValue: (value: string) => void;
+  value: string;
```

Comment:
This may be better called initialValue, value implies that if I pass value and then change it, it will change in component too. But thats probably already established and in many places in the framework, so we will probably leave it as it is?

Reply:
left as it is. added jsdoc comments